src/components/SecondaryPanes/XHRBreakpoints.js

```diff
@@ -242,6 +271,39 @@ class XHRBreakpoints extends Component<Props, State> {
     );
   };
 
+  renderMethodOption = method => {
+    return (
+      <option key={method} value={method}>
+        {method}
+      </option>
+    );
+  };
+
+  renderMethodSelectElement = () => {
+    const methods = [
```

Comment:
We can declare this as a const at the top of the file so we aren't rebuilding this array each time this method is called.

Reply:
Fixed!